api_v3/lib/types/partner/KalturaPartner.php

```diff
@@ -469,6 +469,12 @@ class KalturaPartner extends KalturaObject implements IFilterable
 	 */
 	public $eventPlatformAllowedTemplates;
 	
+	/**
+	 * Is password restriction feature on FirstName/LastName/Email enabled
+	 * @var bool
+	 */
+	public $passwordRestrictionDisabled;
```

Comment:
put this option around line 441 next to the other password options
if no one is going to change it from the api add @readonly
no need for the descriptoin
on api better have it in the oposite direction - name like $allowDeafultPasswordRestrictions set to true by deafult